src/cluster/managers/documentation/CommandDocumentationManager.ts

```diff
@@ -88,6 +88,31 @@ export class CommandDocumentationManager extends DocumentationTreeManager {
         };
     }
 
+    #listCommandNames(names: readonly string[]): string {
+        const charLimit = discord.getLimit('embed.field.value') - 6;
```

Comment:
`- 6` ? magic number aaaaaaaaaaaaaaaaa

Reply:
6 comes from the padding of 3 backticks added to each side of the string, guess I could use a .length instead though

Reply:
Ive refactored this entire method to be more readable, thoughts?